app/controllers/application_controller.rb

```diff
@@ -13,18 +13,22 @@ class ApplicationController < ActionController::Base
   # Custom Error Methos
   module SnapException
     class AccessDenied < StandardError; end
-  end 
+  end
 
   # Clean Error Handler Methods
   # Use these in place of handling errors individually.
-  def access_denied # Equivalent 401
-    raise SnapException::AccessDenied.new('You don\'t have permission to view this.')
+  def access_denied(msg = nil)  # Equivalent 401
+    default = 'You don\'t have permission to view this.'
+    msg = msg || default
+    raise SnapException::AccessDenied.new(msg)
   end
-  
-  def item_not_found # Equivalent 404
-    raise ActiveRecord::RecordNotFound.new('Not Found')
+
+  def item_not_found(msg = nil) # Equivalent 404
+    default = 'This is Could Not Be Found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.